src/services/GraphInit.jl

```diff
@@ -227,46 +227,62 @@ function initManual!( variable::DFGVariable,
   setValKDE!(variable, ptsArr, true, solveKey=solveKey)
   return nothing
 end
-function initManual!( dfg::AbstractDFG, 
+function initVariable!( dfg::AbstractDFG, 
                       label::Symbol, 
                       belief::ManifoldKernelDensity,
                       solveKey::Symbol=:default;
                       dontmargin::Bool=false,
                       N::Int=getSolverParams(dfg).N  )
   #
   variable = getVariable(dfg, label)
-  initManual!(variable, belief, solveKey, dontmargin=dontmargin, N=N)
+  initVariable!(variable, belief, solveKey, dontmargin=dontmargin, N=N)
   return nothing
 end
-function initManual!( dfg::AbstractDFG, 
+
+
+function initVariable!( dfg::AbstractDFG, 
                       label::Symbol, 
                       usefcts::Vector{Symbol},
                       solveKey::Symbol=:default;
-                      dontmargin::Bool=false,
-                      N::Int=getSolverParams(dfg).N )
+                      N::Int=getSolverParams(dfg).N,
+                      kwargs... )
   #
   pts = predictbelief(dfg, label, usefcts, solveKey=solveKey)[1]
   vert = getVariable(dfg, label)
   Xpre = manikde!(getManifold(getVariableType(vert)), pts )
-  initManual!(vert, Xpre, solveKey, dontmargin=dontmargin, N=N )
+  initVariable!(vert, Xpre, solveKey; N, kwargs... )
   # setValKDE!(vert, Xpre, true, solveKey=solveKey)
   # return nothing
 end
 
 
-function initManual!( dfg::AbstractDFG, 
-                      sym::Symbol, 
-                      pts::AbstractVector{P}, 
-                      solveKey::Symbol=:default;
-                      bw=nothing ) where {P}
+function initVariable!( vari::DFGVariable, 
+                        pts::AbstractVector{P}, 
+                        solveKey::Symbol=:default;
+                        bw=nothing ) where {P}
   #
-  var = getVariable(dfg, sym)
-  M = getManifold(var)
-  pp = manikde!(M, pts, bw=bw)
-  initManual!(var,pp, solveKey)
+  # specializations to support generic case of Tuple rather than ProductRepr or ArrayPartition inputs
+  # TODO ArrayPartition inputs
+  _prodrepr(pt) = pt
+  _prodrepr(pt::Tuple) = Manifolds.ProductRepr(pt...)
```

Comment:
@jim-hill-r , so the problem was we are actually using `Manifolds.ProductRepr` to represent on-manifold points.  This is slowly converting to a more standardized format with wider JL community as `ArrayPartition`.  To simplify all that, I making the assumption that if we are reserving `Tuple[...]` of points as a API shorthand which must to automatically converted to `ProductRepr[...]`.  We can resolve this once the `ArrayPartition` change has run it's course:
- #1537 

cc @Affie